core/src/main/java/org/mule/api/security/revocation/RevocationCheck.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.api.security.revocation;
+
+public class RevocationCheck
+{
+    private StandardRevocationCheck standardRevocationCheck;
```

Comment:
Review my comment regarding this. These should all be types of revocation that return appropriate configurations.